src/util/taproot.rs

```diff
@@ -695,13 +710,17 @@ impl ControlBlock {
         })
     }
 
-    /// Obtain the size of control block. Faster and more efficient than calling
-    /// serialize() followed by len(). Can be handy for fee estimation
+    /// Returns the size of control block. Faster and more efficient than calling
+    /// `Self::serialize()` followed by `Self::len()`. Can be handy for fee estimation.
```

Comment:
This is meant to `Self::serialize().len()` instead of `Self::len()`

Reply:
Of course, thanks.